docs/design/README.md

```diff
@@ -539,15 +539,21 @@ the array may be deduced, it may be omitted, as in:
 
 ```carbon
 var i: i32 = 1;
+// [i32;N] [i32;N] is an array representing N values, the numerical type of each value is the data type of i32, the size of the array is determined, and new elements cannot be added, nor can the elements of the array be reduced.
```

Comment:
This information seems like it should be covered in the ordinary text of the document (if it isn't already), rather than in a comment within a code block.

Reply:
I don't see this information in normal text in other documents, if there is, please ignore it.

Reply:
I'm having trouble understanding the text you have here, but I think the paragraph above is trying to convey the same ideas. Feel free to suggest additions or clarifications in that paragraph, though.